examples/sprites/tests/bindings/test_sprites.kts

```diff
@@ -11,3 +11,11 @@ assert( s.getPosition() == Point(1.0, 2.0) )
 
 s.moveBy(Vector(-4.0, 2.0))
 assert( s.getPosition() == Point(-3.0, 4.0) )
+
+s.destroy()
```

Comment:
I'm wondering where we'd put documentation for this, so consumers know they need to call the `destroy` function to do the deallocation. Thoughts?

Reply:
Oh that's a really interesting question. 

For now, I've documented in the generated section, however, we do need a better solution for this.